pkg/config/config.go

```diff
@@ -71,6 +71,8 @@ type CniConfig struct {
 	NetworkPluginBinDir string `toml:"bin_dir" json:"binDir"`
 	// NetworkPluginConfDir is the directory in which the admin places a CNI conf.
 	NetworkPluginConfDir string `toml:"conf_dir" json:"confDir"`
+	// NetworkPluginMaxConfNum is the max number of plugin config file will load
```

Comment:
// NetworkPluginMaxConfNum (default = 1) is the max number of plugin config files that
// will be loaded from the cni config directory. Set the value to 0 to load all config files (no arbitrary limit).

Reply:
And say something like that in the config.md.